pages/registration/student/index.page.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+import { StudentRegistrationContainer } from "@/modules/Registration/containers/StudentRegistration";
```

Comment:
```suggestion
import { StudentRegistrationContainer } from "@/modules/Registration/containers/StudentRegistrationContainer";
```

Let''s be consistent with the naming